projects/packages/installer/src/class-plugin.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Main class file for the Composer plugin that will implement a custom installer for Jetpack
+ * packages.
+ *
+ * @see https://getcomposer.org/doc/articles/custom-installers.md
+ * @package automattic/jetpack-installer
+ * */
+
+namespace Automattic\Jetpack\Installer;
+
+use Composer\Composer;
+use Composer\IO\IOInterface;
+use Composer\Plugin\PluginInterface;
+
+/**
+ * This class is the entry point for the installer plugin. The Composer
+ * installation mechanism registers the plugin by calling its activate method.
+ */
+class Plugin implements PluginInterface {
+
+	/**
+	 * Activates the installer plugin at installation time.
+	 *
+	 * @param Composer    $composer the Composer global instance.
+	 * @param IOInterface $io the IO interface global instance.
+	 */
+	public function activate( Composer $composer, IOInterface $io ) {
+		$installer = new Manager( $io, $composer );
+		$composer->getInstallationManager()->addInstaller( $installer );
+	}
+
+	/**
+	 * Deactivates the installer plugin.
+	 */
+	public function deactivate() {
+		// TODO: implement.
```

Comment:
Looking at https://github.com/composer/installers/blob/main/src/Composer/Installers/Plugin.php, it looks like we'd want line 29 to set `$this->installer` so then here we can do
```suggestion
		$composer->getInstallationManager()->removeInstaller( $this->installer );
```

Reply:
I'm not sure how to test this, but it seems to be working when I do `composer remove`, thanks!